fleet_booking/README.rst

```diff
@@ -0,0 +1,17 @@
+===============================
+ Custom system for car renting
+===============================
+
+Adds departments in fleet module.
```

Comment:
Only? What else?

Specify that departments is based on build-in module (which one?)